orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/admin/web/QueueAdminController.kt

```diff
@@ -15,22 +15,27 @@
  */
 package com.netflix.spinnaker.orca.q.admin.web
 
+import com.netflix.spinnaker.orca.q.StartWaitingExecutions
 import com.netflix.spinnaker.orca.q.admin.HydrateQueueCommand
 import com.netflix.spinnaker.orca.q.admin.HydrateQueueInput
 import com.netflix.spinnaker.orca.q.admin.HydrateQueueOutput
+import com.netflix.spinnaker.q.Message
+import com.netflix.spinnaker.q.Queue
 import org.springframework.web.bind.annotation.RequestMapping
-import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RestController
 import java.time.Instant
 import javax.ws.rs.QueryParam
 
 @RestController
 @RequestMapping("/admin/queue")
 class QueueAdminController(
-  private val hydrateCommand: HydrateQueueCommand
+  private val hydrateCommand: HydrateQueueCommand,
+  private val queue: Queue
 ) {
 
-  @RequestMapping(value = ["/hydrate"], method = [(RequestMethod.POST)])
+  @PostMapping(value = ["/hydrate"])
   fun hydrateQueue(
```

Comment:
I should just delete this endpoint... it's not possible without making executions immutable (Mark, do you uh, wanna get crazy sometime this year? 😆 )